src/Blocks/custom/custom-data/custom-data-editor.scss

```diff
@@ -0,0 +1,20 @@
+.es-block-custom-data {
+	// Transparent overlay over the forms so the user can't click on them from the admin screen.
+	&::after {
+		content: '';
+		background-color: transparent;
+		width: 100%;
+		height: 100%;
+		position: absolute;
+		left: 0;
+		top: 0;
```

Comment:
What you can add to reassure the non-clicking ability is 
`pointer-events: all;` this should ensure to receive all of the events over the after component.

Reply:
I will implement this thx